### ISSUE TITLE
Insert Special Characters

### ISSUE THREAD

**Issue by LoganJFisher** [JerryI/wolfram-js-frontend#212]
**Is your feature request related to a problem? Please describe.**
There currently is no convenient way to use various symbols like Greek letters and various math symbols.

**Describe the solution you'd like**
I would appreciate a menu, like is found in Mathematica, which allow you to select from various sets of special characters.

**Additional context**
![sizrTi4](https://github.com/JerryI/wolfram-js-frontend/assets/63993409/aca17f8c-4af9-49ff-9f19-3c24d7654792)

**Comment by JerryI** [comment]
A palette window is a good option. For now some greek symbols can be entered using ESC key and first few letters, then it can be selected from the autocomplete menu

**Comment by JerryI** [comment]
We added it in a form of a snippet (in a new version of wljs-snippets)

![image](https://github.com/JerryI/wolfram-js-frontend/assets/4111822/89e77cc4-8721-4ac3-b021-14a7babb60c2)

It works similar to `Insert Matrix`, `Import Data` and etc. Just type in the command palette `Glyphs toolbar` and a helper window will appear in a form of a regular cell. 

I added basic greek symbols, fractions, and etc (actually better to use shortcuts for them, i.e. `Ctrl+/`, `Ctrl+2`, `Ctrl+7`...). They can work not only in Wolfram Language, but also in `.html`, `.wlx` and `.md`. This snippet automatically detects the context and uses the corresponding strategy. 

PS: To get a new version: wait 2-3 days (typical  this is a time interval for a package manager to check updates) or call `Check updates` from the settings menu and a new version will be fetched automatically.
